code/datums/helper_datums/teleport.dm

```diff
@@ -180,3 +180,36 @@
 
 /proc/get_teleport_turf(turf/center, precision = 0)
 	return safepick(get_teleport_turfs(center, precision))
+
+/proc/wizarditis_teleport(var/mob/living/carbon/affected_mob)
+	var/list/theareas = get_areas_in_range(80, affected_mob)
+	for(var/area/space/S in theareas)
+		theareas -= S
+
+	if(!theareas||!theareas.len)
+		return
+
+	var/area/thearea = pick(theareas)
+
+	var/list/L = list()
+	for(var/turf/T in get_area_turfs(thearea.type))
+		if(T.get_virtual_z_level() != affected_mob.get_virtual_z_level())
+			continue
+		if(T.name == "space")
+			continue
+		if(!T.density)
+			var/clear = 1
+			for(var/obj/O in T)
+				if(O.density)
+					clear = 0
+					break
+			if(clear)
+				L+=T
+
+	if(!L)
+		return
+
+	if(do_teleport(affected_mob, pick(L), channel = TELEPORT_CHANNEL_MAGIC, no_effects = TRUE))
+		affected_mob.say("SCYAR NILA [uppertext(thearea.name)]!", forced = "wizarditis teleport")
+
+	return
```

Comment:
Since this is still copy and pasted code, convert the wizard spell to use this too (they are the same)

Reply:
I... already did that?

![image](https://user-images.githubusercontent.com/9547572/179417989-ae6d4eb0-964c-427a-af47-7bd444ea7ce4.png)

This is a change to the disease

![image](https://user-images.githubusercontent.com/9547572/179418011-726090e3-2981-4ab6-a620-3c5f7477c988.png)

This is to the symptom.

Reply:
I meant convert the actual wizard spell